controllers/comments-by-id-controller.js

```diff
@@ -8,4 +9,16 @@ exports.getCommentsById = (request, response, next) => {
     .catch((err) => {
         next(err)
     })
+};
+
+exports.postCommentsById = (request, response, next) => {
```

Comment:
Be careful with plurals in names! Does this post comments or a comment - we need to be clear with naming as other devs or your future self may get confused. Your model is singular (which I think is more acurate) 👍

Reply:
Ah yes makes sense!